language/vm/src/errors.rs

```diff
@@ -335,6 +335,10 @@ pub enum VMInvariantViolation {
     InternalTypeError,
     #[fail(display = "Event key is not 32 byte")]
     EventKeyMismatch,
+    #[fail(display = "Execution stack overflow")]
+    ExecutionStackOverflow,
+    #[fail(display = "Call stack overflow")]
+    CallStackOverflow,
```

Comment:
I am not sure those should be VMInvariantViolations but @tzakian is reworking the error story so I just want to flag it to him